npm/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@unsplash/intlc",
+  "version": "0.2.0",
+  "description": "intlc wrapper that makes it available as a local dependency",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "postinstall": "./dist/index.js"
+  },
+  "files": [
+    "dist/index.js"
+  ],
+  "author": "Unsplash <dev@unsplash.com>",
+  "devDependencies": {
+    "@types/download": "^8.0.1",
+    "@types/node": "^17.0.12",
+    "fp-ts": "^2.11.8",
+    "io-ts": "^2.2.16",
+    "typescript": "^4.5.5"
+  },
+  "peerDependencies": {
+    "fp-ts": "^2.11.8",
```

Comment:
This might not be desirable if we open source this npm package as well. Or perhaps there is no need to do that? Having `intlc` might be enough?

Reply:
Worth testing, we don't want that requirement. If needs be we could bundle fp-ts into the script, rather than leave an import of the module, assuming that's not already taking place.

Reply:
For the moment I've left a note in the readme. We can adjust the flow if we want later https://github.com/unsplash/intlc/pull/58/commits/42f1249abe01d637f06b2d52d1a455be3e957ef5